demo/nerdlets/nr1-community-demo-nerdlet/pages/EmptyState/examples/basic.js

```diff
@@ -28,6 +28,7 @@ export default class EmptyStateBasicDemo extends React.Component {
     console.log('You clicked the empty state buttun!');
   }}
   buttonUrl="https://google.com/"
+  featuredImage=""
```

Comment:
While I think we _should_ have an example of this prop included in an example, I don't think this should be included in the basic example. Of course, there is no "advanced" example for the empty state, but I'll go create an issue for that because at this point there probably should be.

Anyway, we can remove the prop from this example and whenever we add an advanced example we can include it over there.